### ISSUE TITLE
Convert to vector : recognition issue

### ISSUE THREAD

**Issue by openanim** [opentoonz/opentoonz#3567]
**Describe the bug**
When using other color than black with a raster level and do a convert to vector, the lines are not recognized with centerline mode.  Sometimes, enbale ink recognition can help.

**To Reproduce**
List steps to reproduce the behavior.
1.draw on raster level with other color than black
2.use convert to vector command in centerline on the level 
3.enable preview on the swatch 
4.no recognition

**Expected behavior**
Recognition in black at least

**Comment by RodneyBaker** [comment]
I confirm the problem here.
I do get recognition with all colors I've tested thus far on toonz raster so a workaround would be to convert to toonz raster and then to vector.

An additional observation:  Yellow colors tend to get converted to green in both raster and toonz raster in the 'convert to vector' process.  I'm not sure if this relates to the rest of the problem but it might be that colors aren't being registered correctly and therefore don't get properly identified for preview and conversion.

**Comment by RodneyBaker** [comment]
A question related to this report...
Where we able to convert from raster to vector with color lines before and if so when did we lose the capability?
My memory says that I've done it many times but perhaps my memory is mistaken.

**Comment by openanim** [comment]
Indeed, I tried with an old version with no difference. I have to confuse it with harlequin. It seems that the enhanced ink recognition command is less efficient in OT. Even if the result is not convincing in harlequin. Recognition seems very irregular in OT. Maybe tag this as an enhancement

**Comment by RodneyBaker** [comment]
Reporter has ghosted themself but the issue reported remains an area of interest.
Recommend inclusion in review of Opentoonz usage of vector (PLI and SVG) formats.
